### PR TITLE
Pre dev

### PR DESCRIPTION
**Pull request by lhalam**

### CODE REVIEW

File: pom.xml
Comment (lhalam) [inline pom.xml:1]: revert change

File: src/test/java/com/historycode/TestRunner.java
Comment (lhalam) [inline src/test/java/com/historycode/TestRunner.java:1]: remove this file

File: src/test/java/com/historycode/HelpUsPageTest.java
Comment (lhalam) [inline src/test/java/com/historycode/HelpUsPageTest.java:1]: remove this file

File: src/main/java/com/historycode/ui/page/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import com.historycode.ui.component.BaseComponent;
+import com.historycode.ui.component.footer.FooterComponent;
+import com.historycode.ui.component.header.HeaderComponent;
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage extends BaseComponent {
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/HomePage.java]: extend BasePage

File: src/main/java/com/historycode/ui/page/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    public HomePage(WebDriver driver, WebElement rootElement) {
+        super(driver, rootElement);
+        PageFactory.initElements(driver, this);
+        PersonCardComponent personCard = new PersonCardComponent(driver, rootElement);
+        NewsCardComponent newsCard = new NewsCardComponent(driver, rootElement);
+        StaticBannerComponent staticBanner = new StaticBannerComponent(driver, rootElement);
+        TeamCardComponent teamCard = new TeamCardComponent(driver, rootElement);
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/HomePage.java]: List

File: src/main/java/com/historycode/ui/page/HomePage.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    public TeamCardComponent(WebDriver driver, WebElement rootElement) {
+        super(driver, rootElement);
+    }
+}
+
+class PartnersComponent extends BaseComponent {
+    public PartnersComponent(WebDriver driver, WebElement rootElement) {
+        super(driver, rootElement);
+    }
+}
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/HomePage.java]: move files

File: src/main/java/com/historycode/ui/page/HomePage.java
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/HomePage.java:1]: create package

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+public class StreetCodeCatalogPage extends BasePage {
+    private static final By CATALOG_CONTAINER_LOCATOR = By.xpath("//div[@class='steetcodeCatalogContainer']");
+
+    @FindBy(xpath = "//h1[@class='streetcodeCatalogHeading']")
+    private WebElement catalogTitle;
+
+    @FindBy(xpath = "//p[@class='streetcodeCatalogCaption']")
+    private WebElement catalogCaption;
+
+    CatalogContainerComponent catalogContainer;
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java]: list

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java:1]: create package

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    public String getCatalogCaption() {
+        return catalogCaption.getText();
+    }
+
+    public List<String> getStreetItemNames() {
+        return catalogContainer.getCatalogNames();
+    }
+}
+
+class CatalogContainerComponent extends BaseComponent {
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java]: move to a new file

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java (hunk 4 above)
Comment (TanyaB777) [reply src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java]: fixed

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java
Comment (TanyaB777) [reply src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java:1]: fixed

File: src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java (hunk 5 above)
Comment (TanyaB777) [reply src/main/java/com/historycode/ui/page/StreetCodeCatalogPage.java]: fixed

File: src/main/java/com/historycode/ui/page/homePage/CarouselComponent.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        super(driver, rootElement);
+        PageFactory.initElements(rootElement, this);
+        this.componentClass = componentClass;
+    }
+
+    public List<T> getCarouselItems() {
+        return carouselItems.stream()
+                .map(e -> {
+                    try {
+                        return componentClass.getConstructor(WebDriver.class, WebElement.class)
```
Comment (lhalam) [inline src/main/java/com/historycode/ui/page/homePage/CarouselComponent.java]: T(WebDriver.class, WebElement.class)

**Pull request merged**